downstream/modules/platform/proc-controller-delete-job-template.adoc

```diff
@@ -6,9 +6,9 @@ Before deleting a job template, ensure that it is not used in a workflow job tem
 
 .Procedure
 
-. Delete a job template by using one of these methods:
-* Select the checkbox next to one or more job templates. Click image:options_menu.png[options menu,15,15] and select btn:[Delete template].
-* Select the required job template, on the *Details* page click image:options_menu.png[options menu,15,15] and select btn:[Delete template].
+. Delete a job template using the following method:
+* Click the {MoreActionsIcon} icon and select the Delete Templateimage:delete-icon.png[Delete Template,15,15] icon.
+* Select the required job template, on the *Details* page click the {MoreActionsIcon}  icon and select and select image:delete-button.png[Delete template,15,15] btn:[Delete template].
```

Comment:
Typo "and select and select" -> "and select"

has an extra "and select" in there. Also, is there an extra space before "the {MoreActionsIcon}  icon"? It looks like it in the code but not in the rich diff.